### PR TITLE
admin page by Ardit

### PR DESCRIPTION
**Pull request by ardithoxha**
Jason and Suchen, could you review my pull request?

### CODE REVIEW

**Comment by jqj5405** [comment]
Well done ardit! On all accounts, this admin page meets all functional expectations and fluid use of CSS. It's refreshing to see how well you adhere to 'angular-folder-structure', you did your research. As you didn't implement services or interface, I should suggest that any interface/services file going forward should be placed with their respective module folder as per the standard. Delving into the components, your implementation is generally straightforward and only deviates slightly from my own version. There is one CL that is inactive but still being called in user-info-panel.ts. In admin-page.ts, I like how you initialize users[0] to display; it works well with the UI design. There are also a few pointers in the HTML and CSS files I came across that may be of scrutiny. I'll leave the general list of issues I noticed below, you can also see my comments in GitHub code editor in this pull request.

user-info-panel.ts
-line 14 => object initial container is unused and doesn't influence @input when using the 'any' type

user-info-panel.css
-line 5 => panel list info is verticle, .container div flex direction is best left as row
-line 6 => panel is left justfied on default

admin-page.html
-line 6 => missing </li>
-line 7 => use (object) true value instead of (string) if-else (readability)
[ngClass]="user.id === userToShow.id ? 'selected': null" to
[ngClass]="{ selected: user.id === userToShow.id }"

admin-page.css
line 18 => change button color (low visibility)
line 44 => no functionality
line 86 => height: 30px; (visibility and selectability)
line 88-90 => obsolete from using justify-content: space-between (line 85)
line 115 => background color covered by buttons

**Review by jqj5405 — COMMENTED** [review]
Suggestions

File: TeamProject/src/app/modules/admin/admin-page/admin-page.component.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+.selected {
+    background-color: #0097e6;
+}
+
+li>div {
+    display: flex;
+    justify-content: space-between;
+}
```
[the remark this answers is not in the document]
Comment (jqj5405) [reply TeamProject/src/app/modules/admin/admin-page/admin-page.component.css]: line 86 => changed to height: 25px

File: TeamProject/src/app/modules/admin/admin-page/admin-page.component.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    border: transparent;
+    border-radius: 5px;
+}
+
+footer {
+    display: flex;
+    justify-content: spread-between;
+    width: 100%;
+    height: 10vh;
+    background-color: #192a56;
```
[the remark this answers is not in the document]
Comment (jqj5405) [reply TeamProject/src/app/modules/admin/admin-page/admin-page.component.css:111]: no change

**Comment by jqj5405** [comment]
good to go

**Comment by SuchenShi** [comment]
In a word, the admin page meets all the requirements asked, I really agree with the way you organize the components. All the implementations are quite clear. The use of observables in the admin page is also clean. I have noticed some other minor things that Jason has already covered. So, good job! This is good to go.

**Pull request merged**